### PR TITLE
Replaced &emsp; with spaces

### PR DESCRIPTION
**Pull request by Poonam-Singh-Bagh**

### CODE REVIEW

File: src/components/Course/Content/ExerciseContent/index.js [hunk 1]
```diff
@@ -25,7 +25,8 @@ ${code}
 const createVisulizeURL = (code, lang, mode) => {
   // only support two languages for now
   let l = lang == "python" ? "2" : "js";
-  let visualizerCode = code && code.replace(/<br>/g, "\n");
+  let replacedCode = code && code.replace(/<br>/g, "\n");
+  let visualizerCode = replacedCode.replace(/&emsp;/g, "    ");
```
Comment (pysaquib) [inline src/components/Course/Content/ExerciseContent/index.js]: @Poonam-Singh-Bagh How many spaces for one `&emsp;` ?

**Review by pysaquib — COMMENTED** [review]
Check

File: src/components/Course/Content/ExerciseContent/index.js (hunk 1 above)
Comment (Poonam-Singh-Bagh) [reply src/components/Course/Content/ExerciseContent/index.js]: 4

Comment (pysaquib) [reply src/components/Course/Content/ExerciseContent/index.js]: @Poonam-Singh-Bagh No. We discussed one space for one `&emsp`.

Comment (Poonam-Singh-Bagh) [reply src/components/Course/Content/ExerciseContent/index.js]: But it's not looking fine

**Pull request merged**